tests/test_semantic_conventions.py

```diff
@@ -228,6 +228,10 @@ def validator(span):
     @bug(library="cpp", reason="language tag not implemented")
     @bug(library="php", reason="language tag not implemented")
     @bug(library="java", reason="language tag implemented but not for all spans")
+    @bug(
+        library="dotnet",
+        reason="language tag implemented but not for all spans. https://datadoghq.atlassian.net/browse/AIT-8735",
+    )
```

Comment:
If you add only reason="AIT-8735" it would be fine

Reply:
And yes, the errors came from this bug, not this PR related.

Reply:
I can see the error in dotnet here: FAILED tests/remote_config/test_remote_configuration.py::Test_RemoteConfigurationExtraServices::test_tracer_extra_services

could you correct the conflicts and launch the CI again?